dom-parser.cabal

```diff
@@ -1,5 +1,5 @@
 name:                dom-parser
-version:             3.0.0
+version:             3.1.0
```

Comment:
But where is support itself? Just version in cabal file is changed.

Reply:
implemented in #12